playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -130,17 +130,44 @@
     - horizon
     - run
 
+- name: Install openstack client
+  package:
+    name: python-openstackclient
+    state: present
+  register: _pkginstall
+  until: _pkginstall is success
+  retries: 5
+  delay: 2
+  tags:
+    - run
 
-# TODO(evrardjp): Use mechanism like glance to re-use component-install.yml code and allow any override.
-# TODO(evrardjp): Check stdout to ensure idempotency. In the meantime skip_ansible_lint
-- name: Setup openstack CLIs
-  command: ./tools/deployment/multinode/010-setup-client.sh
-  args:
-    executable: /bin/bash
+- name: Create /etc/openstack directory
+  file:
+    path: /etc/openstack
+    state: directory
+    owner: root
+    group: root
+    mode: 0755
+  tags:
+    - run
+
+- name: Save default clouds.yaml file
+  copy:
```

Comment:
I would use template: here, so that we can change it later with a few overrides, should that path be pursued.

Reply:
ok, I'll adapt it